frontend/app/routes/$lang/_protected/personal-information/email-address/edit.tsx

```diff
@@ -45,6 +46,8 @@ export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
 });
 
 export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  featureEnabled('edit-personal-info');
```

Comment:
Add the same check in the action.

Reply:
Added in all `action(..)` [here](https://github.com/DTS-STN/canadian-dental-care-plan/pull/1802/commits/e435f782344918d284c268453c1fbf299a395013).